src/config/topics.js

```diff
@@ -9,6 +10,10 @@ import defaultBaseLayers, {
   netzkarteShowcases,
   parks,
   stuetzpunktbahnhoefe,
+  constrUnterhalt,
+  constrAusbau,
+  constrSingleLayer,
+  constrClusterLayer,
```

Comment:
I'm wondering if it's possible to only add one layer:

```
aConstructionLayer = {
     olLayer: ol.layer.Group({
         layers: [  ol.layer.Cluster, ol.layer.vector ]  //  the 2 layers are based on the vector source
     }),
     children: [
          constrUnterhalt,
          constrAusbau,
     ],
     properties: {
        hideInlegend:true
     }
}
```

What do you think ?  I don't know if it's better or not.

Reply:
I will check this.

Reply:
Is your goal to save lines in layers.js?

I don't understand exactly. We still need to configure the `minResolution` and `maxResolution` as well as the GeoJSON loading for both layers. And `ol.layer.Cluster` does not exist. What exactly do you mean?

In order to save lines, we could
* change the `Layer` in `react-spatial` to accept `children` when initializing, see [line](https://github.com/geops/react-spatial/blob/master/src/layers/Layer.js#L37)
* add the "group" layers to the `ConstructionLayers`

```javascript
export const constrSingleLayer = new ConstructionLayer({
  name: 'ch.sbb.construction.single',
  key: 'ch.sbb.construction.single',
  visible: true,
  properties: {
    hideInLegend: true,
    popupComponent: 'ConstructionPopup',
  },
  children: [
    constrUnterhalt,
    constrAusbau,
  ],
  maxResolution: 305.748113141,
});
```

(and the same for constrClusterLayer)

Reply:
yes ol.layer.Cluster doesn't exist. it was a shortcut to an ol layer with a cluster source.
I just meant, can we use a common vector source for both layers ( cluster and the vector) ?  so the data will be loaded only once.

Reply:
The goal is to create one react-spatial laye instead of 2 but maybe it's not possible.

Reply:
and yes react-sptial should be able to manage children from the constructor.

Reply:
> and yes react-sptial should be able to manage children from the constructor.

@oterral this would require changes in react-spatial: [PR](https://github.com/geops/react-spatial/pull/339)